packages/core/src/components/screens/create-incident/CreateIncidentScreen.tsx

```diff
@@ -72,64 +72,66 @@ export const CreateIncidentScreen = () => {
     };
 
     return (
-        <ScrollView contentInsetAdjustmentBehavior="automatic">
-            <Spacer />
-            <Cell>
-                <View style={styles.topTextContainer}>
-                    <Typography variant={"h1"}>Create ticket in ServiceNow</Typography>
-                    <Typography group={"paragraph"} variant={"body_short"}>
-                        We collect information about your device as part of our feedback process. By
-                        submitting, you agree to share the following information:
-                    </Typography>
-                </View>
-                <UserInfo infoType={"User"} infoValue={account?.username} />
-                <UserInfo infoType={"Device Brand"} infoValue={Device.brand} />
-                <UserInfo infoType={"Device"} infoValue={Device.deviceName} />
-                <UserInfo infoType={"Operating System"} infoValue={Device.osVersion} />
-                <UserInfo infoType={"Time Zone"} infoValue={Localization.timezone} />
-                <UserInfo infoType={"Area"} infoValue={Localization.locale} />
-                {ticketNumber && (
-                    <View style={[styles.popupBox, styles.popupSuccess]}>
-                        <Typography>{`Ticket number: ${ticketNumber}`}</Typography>
-                    </View>
-                )}
-                {error && (
-                    <View style={[styles.popupBox, styles.popupDanger]}>
-                        <Typography>
-                            {/* eslint-disable-next-line @typescript-eslint/no-base-to-string,@typescript-eslint/restrict-template-expressions -- this rule sucks */}
-                            {`An error occurred creating your ticket: ${error}`}
+        <KeyboardAvoidingView behavior="padding" keyboardVerticalOffset={115}>
```

Comment:
The `keyboardVerticalOffset` value seems a bit arbitrary. Does this solution work on all devices? iPhone 8, iPad mini, iPad Pro etc

Reply:
This is what we are using in notifications at least, and it seems to work fine for all the devices I have tried. But I will test a bit and verify that it works here as well 👍

Reply:
I tested some iphones and ipads and the styling was a bit weird, but I found a fix by adding `flex:1` to the `KeyboardAvoidingView`. Now it looks good on the two ipads and the two iphones 👍